### PR TITLE
update readme and ssh keys paths

### PR DESCRIPTION
**Pull request by leena-jawale**

### CODE REVIEW

**Comment by sharvarikhedkar** [comment]
should we also update the readme to add the steps to ssh into vm instances?  @leena-jawale

**Comment by leena-jawale** [comment]
> should we also update the readme to add the steps to ssh into vm instances? @leena-jawale

I think this is sufficient https://github.com/ansible-content-lab/azure_terraform_deployment?tab=readme-ov-file#installing-red-hat-ansible-automation-platform

File: README.md [hunk 1]
```diff
@@ -98,7 +100,7 @@ Confirm to create infrastructure or pass in the `-auto-approve` parameter.
 At this point you can ssh into one of the controller nodes and run the installer. The example below assumes the default variables.tf values for `infrastructure_admin_username` and `infrastructure_admin_ssh_private_key_filepath`. 
 
 ```bash
-ssh -i ~/.ssh/id_rsa azureuser@<controller-public-ip> 
+ssh -i <ssh-private-key-path> azureuser@<controller-public-ip>
```
Comment (chrisahl) [inline README.md:103]: Not sure why you changed this since it shows above the value is based on the variables.tf values

Comment (leena-jawale) [reply README.md]: reverted

File: modules/vm/variables.tf [hunk 2]
```diff
@@ -139,13 +139,11 @@ variable "subnet_id" {
 variable "infrastructure_admin_ssh_public_key_filepath" {
   description = "Public ssh key file path."
   type = string
-  default = "~/.ssh/id_rsa.pub"
```
Comment (sharvarikhedkar) [inline modules/vm/variables.tf:142]: I think we should keep the default value for keys, and give user a way to provide another key through -var if they want to

Comment (sharvarikhedkar) [reply modules/vm/variables.tf:142]: removing default will just cause more prompts for the user to fill while doing `terraform apply`, which could be annoying for the user at times

Comment (leena-jawale) [reply modules/vm/variables.tf:142]: defaults exists in the root variables.tf, no need to provide default again inside vm module

Comment (chrisahl) [reply modules/vm/variables.tf:142]: The default is in the top level variables.tf.  Does it hurt to leave it here too?

Comment (leena-jawale) [reply modules/vm/variables.tf:142]: I need to check that

**Review by chrisahl — APPROVED** [review]
My deployment worked.

**Pull request merged**